pkg/cmd/cli/cmd/set/buildhook.go

```diff
@@ -139,7 +139,7 @@ func (o *BuildHookOptions) Complete(f *clientcmd.Factory, cmd *cobra.Command, ar
 	o.Cmd = cmd
 
 	mapper, typer := f.Object()
-	o.Builder = resource.NewBuilder(mapper, typer, resource.ClientMapperFunc(f.ClientForMapping), kapi.Codecs.UniversalDecoder()).
+	o.Builder = resource.NewBuilder(mapper, f.CategoryExpander(), typer, resource.ClientMapperFunc(f.ClientForMapping), kapi.Codecs.UniversalDecoder()).
```

Comment:
@juanvallejo you're in the list here: https://github.com/openshift/origin/issues/14647